### PR TITLE
feat!: minimum supported webpack version is 5.0.0

### PR DESCRIPTION
**Pull request by snitin315**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **documentation update**
- [ ] **typo fix**
- [x] **metadata update**

### Motivation / Use-Case

Drop webpack v4

### Breaking Changes

minimum supported webpack version is 5.0.0

### CODE REVIEW

**Review by alexander-akait — COMMENTED** [review]
Looks like we remove more than expected? Can you check?

File: src/utils/setupOutputFileSystem.js [hunk 1]
```diff
@@ -17,27 +15,9 @@ function setupOutputFileSystem(context) {
...
-      );
-    }
-
-    // Todo remove when we drop webpack@4 support
-    // @ts-ignore
-    if (typeof outputFileSystemFromOptions.mkdirp !== "function") {
-      throw new Error(
-        "Invalid options: options.outputFileSystem.mkdirp() method is expected"
-      );
-    }
```
Comment (snitin315) [inline src/utils/setupOutputFileSystem.js:33]: @alexander-akait Do we need this with webpack v5?

Comment (snitin315) [reply src/utils/setupOutputFileSystem.js:33]: Otherwise, I'll remove test cases for this as well, which are currently failing.

Comment (alexander-akait) [reply src/utils/setupOutputFileSystem.js:33]: If this tests for webpack  - Yes

Comment (alexander-akait) [reply src/utils/setupOutputFileSystem.js:33]: We have a lot of integration tests, so we will catch if it will have problems

**Pull request merged**